ui/src/containers/Drawer/Drawer.tsx

```diff
@@ -1,9 +1,16 @@
-import React, { useEffect, useRef } from 'react'
+import React, { useRef, useEffect, useState } from 'react'
+import { Redirect } from 'react-router-dom'
+import { useQuery, useMutation } from '@apollo/client'
 import { useHistory } from 'react-router-dom'
 import Konva from 'konva'
 import { IFrame } from 'konva/types/types'
 import { Layer } from 'konva/types/Layer'
 import Wheel from './components/Wheel'
+import { winnerVar } from '@src/config/apolloClient'
+import { CUR_AWARD_QUERY } from '@src/query/award'
+import { CHOOSE_WINNER } from '@src/query/winner'
+import { GetCurrentAward } from '@src/__generated__/GetCurrentAward'
+import { ChooseWinner } from '@src/__generated__/ChooseWinner'
```

Comment:
I noticed the folder name '__generated__'? Is it created by graphQL?

Reply:
Yes, it created by` apollo-tooling`  to gen the typescript definition for query.
https://www.npmjs.com/package/apollo#apollo-clientcodegen-output

strange that the GitHub page remove this command now @@,I saw it few day before , maybe it's changing  sth